build/scripts/scripts.fsproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>

```

Comment:
FYI @gregkalapos - this change is not required. However, I had repeated ([Apple Silicon-related](https://stackoverflow.com/questions/68204296/mac-m1-chip-issues-with-rosetta-attachment-of-code-signature-supplement-failed)) problems when running the `./build agent-zip` command, so I just kept it in.